src/parsers/sample-models.hpp

```diff
@@ -20,6 +20,7 @@
 #define __se3_sample_models_hpp__
```

Comment:
The two new models are never sampled, but there is the name sample in the title which may lead to some confusions. Can you change it for something like {basic, simple} models?

Reply:
I guess "sample" is used in the sense "specimen serving as example"

Reply:
I guess "sample" is used in the sense "specimen serving as example"

Reply:
But, I would make things clear and use simple or basic terms compared to buildHumanoidSimple which in fact a sampled model.

Reply:
But, I would make things clear and use simple or basic terms compared to buildHumanoidSimple which in fact a sampled model.